src/wpool/mongoose_wpool.erl

```diff
@@ -152,7 +149,7 @@ start(PoolType, HostType, Tag, PoolOpts) ->
             pool_opts(), conn_opts()) -> start_result().
 start(PoolType, HostType, Tag, PoolOpts, ConnOpts) ->
     {Opts0, WpoolOptsIn} = proplists:split(PoolOpts, [strategy, call_timeout]),
-    Opts = lists:append(Opts0) ++ maps:to_list(default_opts(PoolType)),
+    Opts = lists:append(Opts0),
```

Comment:
Good, this got simplified later 😆

Reply:
Yeah, but it still made sense to keep the `PoolOpts` a list, because that's what `wpool` expects.